justfile

```
@@ -28,11 +28,12 @@ test-all: test-add-chain test-superchain test-validation
 test-add-chain:
 	# We separate the first test from the rest because it generates artifacts
 	# Which need to exist before the remaining tests run.
-	TEST_DIRECTORY=./add-chain go run gotest.tools/gotestsum@latest --format testname -- -run TestAddChain_Main
-	TEST_DIRECTORY=./add-chain go run gotest.tools/gotestsum@latest --format testname -- -run '[^TestAddChain_Main]'
+	TEST_DIRECTORY=./add-chain go run gotest.tools/gotestsum@latest --format testname -- -count=1 -run TestAddChain_Main
+	TEST_DIRECTORY=./add-chain go run gotest.tools/gotestsum@latest --format testname -- -count=1 -run '[^TestAddChain_Main]'
 
 # Test all Go code in the superchain module
 test-superchain:
+	just clean-add-chain
```

We are using this syntax elsewhere to declare one just command as a dependency in the definition of another:
```suggestion
test-superchain: clean-add-chain
```